### DISCUSSION TITLE
Research papers?

### DISCUSSION THREAD

**Discussion by kirawi** [microsoft/verona#530] [Q&A] [answered]
Hi, I was wondering there were any research papers that inspired Verona, or maybe even any papers written about the language? I'm just fascinated by what the underlying principles of the language may be.

**Comment by mjp41** [comment]
We have some work in preparation, but nothing is published yet.

**Comment by sophiaIC** [comment, marked as answer]
There exist two papers now

A paper at OOPSLA 2023, about the concurrency model in Verona, https://dl.acm.org/doi/10.1145/3622852.

And another paper at OOPSLA 2023, about types and memory management in Verona, https://dl.acm.org/doi/10.1145/3622846
